examples/layouting.jl

```diff
@@ -486,4 +486,105 @@ end
 
     end
 
+    @cell "Ellipse markers and arrows" [layout, "2d"] begin
+
+        # here we plot some grouped coordinates, find their mean and variance which
+        # we plot as ellipses, plot their actual means, and some additional details. 
+
+        using DataFrames, Distributions, Colors
+        using MakieLayout
+        import AbstractPlotting:px
+
+        # Here we create `ngrp` groups of `n` coordinates. The mean coordinates of 
+        # the groups are distributed equally around the unit circle. The coordinates
+        # are drawn from a normal distribution (with standard deviation σ). 
+        ngrp = 4
+        groups = Symbol.(Iterators.take('a':'z', ngrp))
+        n = 5
+        σ = [0.2, 0.4]
+        intended = (; Dict(g => Point2f0(Iterators.reverse(sincos(θ))...) for (g, θ) in zip(groups, range(0, step = 2π/ngrp, length = ngrp)))...)
+        df = DataFrame(group = Symbol[], x = Point2f0[])
+        for g in groups
+            d = MvNormal(intended[g], σ)
+            for _ in 1:n
+                push!(df, (group = g, x = rand(d)))
+            end
+        end
+
+        # assign colors to each group
+        colors = Dict(zip(groups, distinguishable_colors(ngrp, [colorant"white", colorant"black"], dropseed = true)))
+
+        # calculate the mean of each group, and the FWHM of a Gaussian fit to the 
+        # data. We later use the mean and FWHM to plot ellipses.
+        ellipses = by(df, :group) do g
+            n = length(g.x)
+            X = Array{Float64}(undef, 2, n)
+            for i in 1:n
+                X[:,i] = g.x[i]
+            end
+            dis = fit(DiagNormal, X)
+            radii = sqrt(2log(2))*sqrt.(var(dis)) # half the FWHM
+            ellipse = (origin = Point2f0(mean(dis)), radius = Vec2f0(radii))
+            (ellipse = ellipse, )
+        end
+
+        # some helper functions
+
+        # return a vector of coordinates of an ellipse
+        mydecompose(origin, radii) = [origin + radii .* Iterators.reverse(sincos(t)) for t in range(0, stop = 2π, length = 51)]
+        # brighten a color 
+        brighten(c, p = 0.5) = weighted_color_mean(p, c, colorant"white")
+        # darken a color
+        darken(c, p = 0.5) = weighted_color_mean(p, c, colorant"black")
+
+        scene, layout = layoutscene(0, fontsize = 10, font = "arial", resolution = (500,400));
```

Comment:
I just realized we don't ship Arial.  We could use Helvetica instead?  https://github.com/JuliaPlots/AbstractPlotting.jl/tree/master/assets/fonts
```suggestion
        scene, layout = layoutscene(0, fontsize = 10, font = "helvetica", resolution = (500,400));
```